pyfarm/jobtypes/core/internals.py

```diff
@@ -292,46 +292,6 @@ class Process(object):
     def __init__(self):
         self.start_called = False
         self.stop_called = False
-        self._stopped_deferred = None
-        self._started_deferred = None
-
-    @property
-    def stopped_deferred(self):
```

Comment:
Could you go over the specifics as for why we're getting rid of these properties?  These are setup to try and ensure it's harder to missorder the setup of start_deferred/stop_deferred.